client/src/main/java/io/pravega/client/stream/impl/ReaderGroupStateManager.java

```diff
@@ -78,6 +79,9 @@
     private final TimeoutTimer acquireTimer;
     private final TimeoutTimer fetchStateTimer;
     private final TimeoutTimer checkpointTimer;
+    @Getter
+    @GuardedBy("this")
+    private String latestDelegationToken;
```

Comment:
The whole point of delegation tokens is that they are never shared and never transmitted without SSL. This is doing both!

Reply:
This is just a local in memory variable. This is extracted from grpc call to the controller and passed on to segmentstore to get the segmentinfo. Both happen on SSL.